float32 theta1
float32 theta2
float32 scalar
---
float32 result